docs/cli/skill_command.md

```
@@ -103,16 +103,16 @@ datus skill remove sql-optimization
 
 ## REPL Equivalents
 
-All `datus skill` subcommands are available in the REPL as `.skill` commands:
+All `datus skill` subcommands are available in the REPL as `/skill` commands:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)